Dockerfile.php7.0-busybox

```diff
@@ -0,0 +1,327 @@
+	cp /usr/lib/libc-client.so.2007e.0 /usr/lib/x86_64-linux-gnu/libc-client.a ; \
+	cd /usr/src/php-7.0.22/; \
+	./configure --build="x86_64-linux-gnu" --enable-fpm --enable-static --disable-shared \
+	--with-fpm-user=www-data --with-fpm-group=www-data --with-config-file-path="$PHP_INI_DIR" \
+	--with-config-file-scan-dir="$PHP_INI_DIR/conf.d" --with-bz2 --enable-bcmath \
+	--enable-calendar --enable-dba --enable-exif --enable-ftp --with-gd --with-gettext \
+	--with-kerberos --with-openssl --with-imap=/usr/lib --with-imap-ssl --with-png-dir=/usr \
+	--with-jpeg-dir=/usr --enable-intl --with-ldap --with-mcrypt --with-mysqli --with-pdo-mysql \
+	--enable-shmop --enable-soap --enable-sockets --enable-sysvmsg --enable-sysvsem \
+	--enable-sysvshm --enable-wddx --with-xmlrpc --with-xsl --enable-zip --enable-mbstring \
+	--enable-mysqlnd --with-curl --with-libedit --with-openssl --with-zlib --with-pcre-regex=/usr \
+	--with-libdir="lib/x86_64-linux-gnu"; \
+	make -j "$(nproc)";
+
+RUN set -ex; \
+	cd /usr/src/php-7.0.22/; \
+	mkdir pecl; \
+	pecl_install.sh memcached; \
+	pecl_install.sh uopz; \
+	pecl_install.sh igbinary; \
+	#pecl_install.sh ssh2; \
+	pecl_install.sh imagick; \
+	pecl_install.sh msgpack; \
+	pecl_install.sh redis; \
+
+	cd /usr/src/php-7.0.22; \
+	mkdir .deploy; \
+	cd .deploy; \
+	mkdir -p var/log/newrelic; \
+	mkdir -p usr/lib; \
+	mkdir -p usr/local/bin; \
+	mkdir -p usr/local/sbin; \
+	mkdir -p usr/local/etc/php/conf.d; \
+	mkdir -p usr/local/include/php/main; \
+	mkdir -p usr/local/lib/php/extensions/no-debug-non-zts-20151012;
+
+RUN set -ex; \
+	\
+	cd /usr/src/php-7.0.22/.deploy; \
+	NR_VERSION="$( \
+		curl --connect-timeout 10 -sS https://download.newrelic.com/php_agent/release/ \
+			| sed -n 's/.*>\(.*linux\).tar.gz<.*/\1/p' \
+	)"; \
+	curl --connect-timeout 10 -o nr.tar.gz -fSL "https://download.newrelic.com/php_agent/release/$NR_VERSION.tar.gz"; \
+	tar -xf nr.tar.gz; \
+	cp $NR_VERSION/agent/x64/newrelic-20151012.so usr/local/lib/php/extensions/no-debug-non-zts-20151012/newrelic.so; \
+	rm -rf newrelic-php5* nr.tar.gz;
+
+RUN set -ex; \
+	\
+	cd /usr/src/php-7.0.22/.deploy; \
+	curl --connect-timeout 10 -o ioncube.tar.gz -fSL "https://downloads.ioncube.com/loader_downloads/ioncube_loaders_lin_x86-64.tar.gz"; \
+	tar -zxvf ioncube.tar.gz; \
+	mkdir -p /usr/lib/php; \
+	cp ioncube/ioncube_loader_lin_7.0.so usr/local/lib/php/extensions/no-debug-non-zts-20151012/ioncube.so; \
+	rm -Rf ioncube*; \
+	{ \
+		echo "zend_extension=ioncube.so"; \
+	} > usr/local/etc/php/conf.d/ioncube.ini
+
+RUN set -ex; \
+	cd /usr/src/php-7.0.22/.deploy; \
+	cp ../sapi/fpm/php-fpm usr/local/sbin/.; \
+	cp ../sapi/cli/php usr/local/bin/.; \
+	cp ../main/php_version.h usr/local/include/php/main/.; \
+	cp /usr/lib/libc-client.so.2007e.0 usr/lib/.; \
+	cp /usr/lib/libmcrypt.so.4.4.8 usr/lib/.; \
+	cp /lib/x86_64-linux-gnu/libz.so.1.2.8 usr/lib/.; \
```

Comment:
Yes. That would be ideal. I am expecting to have to iterate on this solution a bit more before it is ready to ship. This section was just shoving in the required files to show it works. Next step is to make it elegant and hopefully smart enough to use a scratch base image.